### DISCUSSION TITLE
Stellarium 23.2 on Debian 12

### DISCUSSION THREAD

**Discussion by MaksimPatrikevitch** [Stellarium/stellarium#3366] [Q&A] [answered]
Hello everyone,
just installed Stellarium 0.22.2 on Debian 12.
I would like to enjoy the last Stellarium 23.2 but can't figure out how to.
Anyone can help ?
Cheers

**Comment by t4saha** [comment, marked as answer]
@MaksimPatrikevitch 
You can build from source code
or
You can download an appimage https://github.com/Stellarium/stellarium/releases/download/v23.2/Stellarium-23.2-qt5-x86_64.AppImage
or
You can use a snap/flatpak package.
